### PR TITLE
corrected styling in profile section

### PR DESCRIPTION
**Pull request by IromIbe**
# Fixes Issue/Linear Ticket

( This could be an existing issue or a linear ticket )

**My PR Closes #issue_number_here** or
**My PR Fixes ID-linear_ticket_number_here**

# Changes proposed

## What were you told to do?
correct some styling bugs noticed on the profile section.

## What did you do?

I worked on the profile.jsx component and profile.module.css 
commented the section where the social icons was displayed on the dom.
made alignment corrections on the name in profile section.
# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

zuri

zuri old

### CODE REVIEW

File: packages/main/src/components/protected/topbar/components/Profile.jsx [hunk 1]
```diff
@@ -202,7 +202,7 @@ const Profile = () => {
           <div className={styles.infoContent}>{currentTime}</div>
         </div>
 
-        <div className={styles.social}>
+        {/* <div className={styles.social}>
```
Comment (billmal071) [inline packages/main/src/components/protected/topbar/components/Profile.jsx:205]: If this is commented out and not needed, you can remove it.

File: packages/main/src/components/protected/topbar/styles/Profile.module.css [hunk 2]
```diff
@@ -247,7 +248,7 @@
 }
 .h3users {
   font-weight: 700;
-  margin-right: -1em;
+  /* margin-right: -1em; */
```
Comment (billmal071) [inline packages/main/src/components/protected/topbar/styles/Profile.module.css:251]: remove if it's fixed.

File: packages/main/src/components/protected/topbar/styles/Profile.module.css [hunk 3]
```diff
@@ -258,7 +259,7 @@
   padding-bottom: 10px;
   position: absolute;
   /* left: 15px; */
-  right: 70px;
+  /* right: 70px; */
```
Comment (billmal071) [inline packages/main/src/components/protected/topbar/styles/Profile.module.css:262]: remove commented out code.

File: packages/main/src/components/protected/topbar/styles/Profile.module.css [hunk 4]
```diff
@@ -257,9 +264,9 @@
   transition: background-color 0.3s cubic-bezier(0.165, 0.84, 0.44, 1);
 }
 .myp {
-  margin: 7px 1px 10px !important;
+  margin: 10px 1px 10px !important;
```
Comment (billmal071) [inline packages/main/src/components/protected/topbar/styles/Profile.module.css:268]: Why use !important? You can look for what's affecting the style and fix it.

**Review by billmal071 — CHANGES_REQUESTED** [review]

**Comment by JulianaSau** [comment]
This profile component should appear just as threads does. It should not sit on top of the message board. Rather, it should be within the right space of the message board. 

Also, why is the profile modal not closing once a user clicks on view profile?

**Comment by IromIbe** [comment]
made corrections as requested by the reviewer

**Review by ickynavigator — APPROVED** [review]

**Comment by IromIbe** [comment]
i've fixed merge conflicts

**Pull request merged**